sideboard/configspec.ini

```diff
@@ -52,18 +60,25 @@ checker.check_skipped_app_config = boolean(default=False)
 [rpc_services]
 ___many___ = string
 
-#[[___many___]]
-#jsonrpc_only = boolean(default=False)
+[[__many__]]
+jsonrpc_only = boolean(default=False)
 
 
 [loggers]
-root = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.error = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.access = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-__many__ = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+root = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.error = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.access = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="CRITICAL")
+__many__ = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="INFO")
```

Comment:
Hmm, good point.  We use different defaults where I work, but in production its all managed by puppet anyway, so I don't think it the default defined here would matter anyway.